mlflow/store/artifact/hdfs_artifact_repo.py

```diff
@@ -248,3 +254,45 @@ def as_pair(config):
         list_of_key_val = [as_pair(conf) for conf in extra_conf.split(',')]
         return dict(list_of_key_val)
     return None
+
+
+def archive_artifacts(hdfs_artifact_repository, dest_path, archive_name):
+    """
+    Use hadoop archive to store all artifacts of a run in a har
+
+    :param artifact_path: String path of run artifacts
+    :param dest_path: String path directory where to create the archive
+    :param archive_name: String name of the archive
+
+    :return: path to the new artifact
+    :note: It overrides existing archive.
+    """
+    # clean existing archive if exists
+    remove_folder(dest_path + '/' + archive_name)
+
+    list_artifacts = " ".join(
+        [file_info.path for file_info in hdfs_artifact_repository.list_artifacts()])
+    cmd = "hadoop archive -archiveName {archive_name} -p {artifact_path} "\
+        "{list_artifacts} {dest_path}".format(
+            archive_name=archive_name, artifact_path=hdfs_artifact_repository.path,
+            list_artifacts=list_artifacts, dest_path=dest_path)
+    print(
```

Comment:
A logger in info or debug would be better than a print.

Reply:
Done